packages/codeeditor/src/editor.ts

```diff
@@ -602,6 +612,8 @@ namespace CodeEditor {
    */
   export
   let defaultConfig: IConfig = {
+    fontFamily: '\'Source Code Pro\', monospace',
```

Comment:
I am -1 on moving this (and almost only this) to having a canonical value in the source code... perhaps leaving these as null such that the theme layer can carry it's preferred font, and these settings are really an end-user override.